themes/tiddlywiki/vanilla/base.tid

```diff
@@ -1792,6 +1791,12 @@ html body.tc-body.tc-single-tiddler-window {
 	width: 100%;
 }
 
+.tc-controlpanel svg.tc-image-button{
+	width: 1em;
+	height: 1em;
```

Comment:
@pmario, I have gone and fixed the class to `tc-control-panel` in the demo http://1954.tiddlyspot.com for review.

The buttons have the very same size as in the dropdowns, so they cannot be **too** small, since that's what the main UI is using as well.